lib/Doctrine/ORM/EntityManager.php

```diff
@@ -242,6 +244,8 @@ public function transactional($func)
 
             throw $e;
         }
+
+        return $this;
```

Comment:
This doesn't make any sense. Not only is this function supposed to return actual data (rather than `$this`) but your change is unreachable code.